src/Symfony/Component/ExpressionLanguage/ExpressionLanguage.php

```diff
@@ -97,6 +97,19 @@ public function parse($expression, array $names)
         return $parsedExpression;
     }
 
+    /**
+     * @param Expression|string $expression The expression to check parse available
+     * @param array|null        $names      When provided null - expression syntax will be checked without check names
```

Comment:
`The list of acceptable variable names in the expression, or null to accept any names`